editorial-communication/cover-letter-submission.Rmd

```diff
@@ -0,0 +1,36 @@
+---
+title: Letter to the Editor
+author: "Not just for programmers: A friendly guide on the benefits of GitHub for accelerating collaborative research in Ecology and Evolution"
+date: "Last compiled on `r format(Sys.time(), '%d %B, %Y')`"
+output:
+  html_document:
+    theme: cosmo 
+    highlight: tango
+    self_contained: true
+---
+```{r include=FALSE}
+knitr::opts_chunk$set(fig.retina = 2)
+```
+Dear Editor,
+
+We would like to submit our manuscript entitled “Not just for programmers: A friendly guide on the versatility of GitHub for accelerating collaborative research in Ecology and Evolution” for consideration to be published in OUR_CHOICE_OF_JOURNAL. 
+Our study provides researchers in ecology and evolution with a critical review on thirteen practical ways they can use Github -- an online, cloud-based service for cooperative and documented code development -- to make their research more collaborative, transparent, and reproducible.
+
+Scientific advancement fundamentally requires researchers to adopt reproducible, collaborative and transparent practices.
+
+Most scientists, including ecologists and evolutionary biologists, have become critically dependent on computational tools in their research.
```

Comment:
Maybe a citation here?

Reply:
Yes, I agree with adding a couple citations to the cover letter to show that we know the state of current GitHub use in EcoEvo